larwirecell/Components/CookedFrameSink.cxx

```diff
@@ -128,22 +129,29 @@ void CookedFrameSink::visit(art::Event & event)
 	    recob::Wire::RegionsOfInterest_t roi(nticks);
 	    roi.add_range(tbin, charge.begin(), charge.begin() + ncharge);
 
-	    // geo::kU, geo::kV, geo::kW
-	    auto wpid = m_anode->resolve(chid);
-	    geo::View_t view;
-	    switch(wpid.layer()) {
-	    case WireCell::kUlayer:
-		view = geo::kU;
-		break;
-	    case WireCell::kVlayer:
-		view = geo::kV;
-		break;
-	    case WireCell::kWlayer:
-		view = geo::kW;
-		break;
-	    default:
-		view = geo::kUnknown;
-	    }
+	    // // geo::kU, geo::kV, geo::kW
+	    // auto wpid = m_anode->resolve(chid);
+	    // geo::View_t view;
+	    // switch(wpid.layer()) {
+	    // case WireCell::kUlayer:
+		// view = geo::kU;
+		// break;
+	    // case WireCell::kVlayer:
+		// view = geo::kV;
+		// break;
+	    // case WireCell::kWlayer:
+		// view = geo::kW;
+		// break;
+	    // default:
+		// view = geo::kUnknown;
+	    // }
+
```

Comment:
Instead of commenting these code out, please just remove it.

Reply:
Okay, it's removed.